### DISCUSSION TITLE
Impedance Pattern Plot

### DISCUSSION THREAD

**Discussion by Sonoton** [edwardkort/WWIDesigner#138] [Q&A] [answered]
Can you describe the basis of the Impedance pattern plot.  Is it the derivative of reactance wrt frequency, and how does it relate to playability.   The reason I ask:    In tuning my instruments, I use the spectrum graph, and walk the impedance of a note into the maximum of the loop gain by modifying hole position or hole size.   I notice that  the resulting tuning table and the impedance pattern plots  often show that the 'predicted'  frequency is shown to be off  the target (desired) frequency, even though the spectral plot shows the actual instrument output will be at the spectral max for that note. 

Could you also describe what the Im(Z) corr column  in the Supplementary Information Tool is telling me.

Thank you.

**Comment by bhp1** [comment, marked as answer]
[This wiki page](https://github.com/edwardkort/WWIDesigner/wiki/Evaluating-Instruments-with-the-Tuning-Table-and-Tuning-Graph#about-the-tuning-graph) explains the tuning graph.  The y-dimension is the ratio X/R calculated by the whistle or flute model.  The line for each note extends from the predicted minimum frequency to the predicted maximum frequency.  In principle, if you use the right breath pressure, you can play the note at any pitch between fmin and fmax.  (The model doesn't say anything about the tone quality of the note anywhere between those extremes, however.)  

The maximum loop gain occurs at fmax.  You could, in principle, play every note at fmax, but I don't think you would want to - for most whistles, it will be a lot of work, and the second octave will sound much louder than the first.  The whistle and flute models assume the player will want to play lower notes closer to fmax and higher notes closer to fmin, with a uniform increase in air speed from the lowest note to the highest note.

As noted on [this wiki page](https://github.com/edwardkort/WWIDesigner/wiki/The-Supplementary-Information-Table#about-the-supplementary-information), Im(Z) corr is used (only) in a procedure for [calibrating reed instruments](https://github.com/edwardkort/WWIDesigner/wiki/Working-with-the-Reed-Study-Model#using-the-calibration-spreadsheet-for-calibration).

**Reply by Sonoton** [reply]
Thank you, this helped quite a bit.
